src/test/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/ParseRequestServiceTest.java

```diff
@@ -132,127 +148,125 @@ void validateUserProfiles_throwsResourceNotFound_someProfilesNotFoundTest() {
     }
 
     @Test
-    void validateJudicialProfilesTest() {
-
-        JudicialProfile profile = TestDataBuilder.buildJudicialProfile("37395", "EMP37395",
-                "Magistrate", "Joe", "Bloggs", "Joe Bloggs", "Miss",
-                "1", "Fee Paid Judiciary 5 Days Mon - Fri", "EMP62506@ejudiciary.net",
-                LocalDateTime.of(2020, 4, 28, 16, 1, 0),
-                LocalDateTime.of(2020, 4, 28, 16, 1, 0),
-                "2020-04-28T16:00:49", "TRUE",
-                Collections.singletonList(TestDataBuilder.buildJPAppointment("84",
-                        "5",
-                        "1351",
-                        "1",
-                        LocalDateTime.of(2020, 4, 28, 16, 1, 0),
-                        LocalDateTime.of(2020, 4, 28, 16, 1, 0))),
-                Collections.singletonList(TestDataBuilder.buildJPAuthorisation("52149")),
-                "Judicial");
-
-        sut.validateUserProfiles(Collections.singletonList(profile),
-                judicialUser,
-                integer,
+    void validateJudicialProfilesTest() throws IOException {
+        objectMapper.registerModule(new JavaTimeModule());
+        JudicialProfile judicialProfile1 =
+                objectMapper.readValue(new File("src/main/resources/judicialProfileSample.json"),
+                        JudicialProfile.class);
+        JudicialProfile judicialProfile2 =
+                objectMapper.readValue(new File("src/main/resources/judicialProfileSample.json"),
+                        JudicialProfile.class);
```

Comment:
just make judicialProfile2 equal to judicialProfile1 instead of reading from the file twice

Reply:
This has been resolved